rmgpy/rmg/model.py

```diff
@@ -1051,7 +1051,7 @@ def prune(self, reactionSystems, toleranceKeepInEdge, maximumEdgeSpecies, minSpe
                 speciesToPrune.append((index, spec))
                 pruneDueToRateCounter += 1
             # Keep removing species with the lowest rates until we are below the maximum edge species size
-            elif numEdgeSpecies - len(speciesToPrune) > maximumEdgeSpecies:
+            elif numEdgeSpecies - len(speciesToPrune) > maximumEdgeSpecies and maxEdgeSpeciesRateRatios[index] < toleranceMoveToCore:
```

Comment:
should we have a warning telling the user to use a higher maximumEdgeSpecies value when numEdgeSpecies - len(speciesToPrune) > maximumEdgeSpecies but maxEdgeSpeciesRateRatios[index] > toleranceMoveToCore?

Reply:
@kehang Let us know when you get this added